### PR TITLE
Add variadic generics version of `Defaults.updates()`

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: Sources/Defaults/Defaults.swift [hunk 1]
```diff
@@ -292,13 +292,64 @@ extension Defaults {
...
+			}
+
+			var observations = [DefaultsObservation]()
+
+			if initial {
+				continuation.yield(getCurrentValues())
+			}
+
+			for key in repeat (each keys) {
+				let observation = DefaultsObservation(object: key.suite, key: key.name) { _, _  in
```
Comment (sindresorhus) [inline Sources/Defaults/Defaults.swift:325]: Note: I'm intentionally not using CompositeDefaultsObservation for simplicity, as it's not really needed here.

File: Sources/Defaults/Defaults.swift [hunk 2]
```diff
@@ -292,13 +292,64 @@ extension Defaults {
...
+				}
+
+				observation.start(options: [])
+				observations.append(observation)
+			}
+
+			let immutableObservations = observations
+
+			continuation.onTermination = { _ in
+				// `invalidate()` should be thread-safe, but it is not in practice.
```
Comment (hank121314) [inline Sources/Defaults/Defaults.swift:336]: Curious about this. I thought that `invalidate()` would be thread-safe with Lock, 
Should we use `DispatchQueue.sync` instead inside the `DefaultsObservation`?

Comment (sindresorhus) [reply Sources/Defaults/Defaults.swift:336]: I'm not 100% sure. I have seen KVO crashes in some of my apps that may have been caused by some threading problems in macOS. I prefer to keep it limited to this code until I can 100% confirm it. This is just a precaution.

Comment (hank121314) [reply Sources/Defaults/Defaults.swift:336]: Got it! Thanks for the explanation!

**Review by hank121314 — APPROVED** [review]

**Pull request merged**